packages/atlas/src/config/reactions.ts

```diff
@@ -0,0 +1,9 @@
+export type ReactionId = 1 | 2 | 3 | 4 | 5
```

Comment:
Maybe we extract it from the object below to avoid duplication? `type ReactionId = keyof REACTION_TYPE`?